Clinical-Genomics/scout
# Hotfix 4.41.1 --DO NOT MERGE into MASTER--

This PR adds a functionality or fixes a bug, and potentially marks a new Scout hot-fix release.

**How to test**:
1. add assessment to variants shared between different cases for the same institute
2. generate general report - should fail
3. apply patch and note report working

E.g.
https://scout-stage.scilifelab.se/cust002/F0016954/51f4367225075f69a48bdefbf859d4e4
https://scout-stage.scilifelab.se/cust002/F0015149/0ea90adad7daee2b19fdc3098a68e0d4
then generate report.

Tested ok on stage. 

We apply semantic versioning.  This constitutes a patch release since it is a small fix to a bug introduced yesterday.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
